CMakeLists.txt

```diff
@@ -11,6 +11,8 @@ project(RDASApp VERSION 1.0.0 LANGUAGES C CXX Fortran )
 
 include(GNUInstallDirs)
 enable_testing()
+#clt for rrfs test
+set(MPIEXEC_MAX_NUMPROCS 120 CACHE STRING "Maximum number of processors")
```

Comment:
why is this set to 120 but then 80 is used in the test?

Reply:
It is the maximum allowed MPI numbers and hence this larger number is used now.

Reply:
I guess I'm asking why is this defined, I'm nervous about putting something like this in at build time unless absolutely needed.

Reply:
@CoryMartin-NOAA   It is because this number I speculate had been defined in the MPI module provided by GDAS setup and that was a smaller number  (40).  While RDAS developers/myself don't have access to those modules,     It should be overridden with a larger number.  This location (in build.sh) to do this  is currently where I choose most transparent to RDAS developers/users.    Any suggestions on a better choice  is welcome.

Reply:
No, that's not defined at all in GDASApp. 40 is the number of processors per node on Hera, and that is what CMake sets it to by default.

Just so I understand, are you saying that without this variable set, CMake will complain if you try to make a ctest with more than 40 processors?

Reply:
Got it, thanks. Let's leave it as you have it for now then.